docs/HowToRunSTDKSample.md

```diff
@@ -0,0 +1,254 @@
+  ...
+==================================================
+6               // Select "STDK".
+
+STDK is selected
+  Copy build environment files
+ 
+  CAUTION!! To download esp_wrover_kit/STDK built image, genromfs should be installed
+  Find details at https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseROMFS.md
+ 
+  Configuration is Done!
+======================================================
+  "Select build Option"
+======================================================
+  "1. Build with Current Configurations"
+  "2. Re-configure"
+  "3. Menuconfig"
+  "4. Build Clean"
+  "5. Build Dist-Clean"
+  "6. Build SmartFS Image"
+  "x. Exit"
+======================================================
+1               // Select "Build with Current Configurations"
+
+...
+######################################
+##          Library Sizes           ##
+######################################
+    .data   .bss    .text    Total
+    356     12560   222108  235024  libexternal.a
+    340     8544    118702  127586  libnet80211.a
+...
+    0   1036    2164    3200    NOLIB
+    24  380     2076    2480    libmm.a
+    4   4   1904    1912    librtc.a
+    0   24  980     1004    libwque.a
+    0   42  800     842     libcore.a
+    0   0   796     796     libboard.a
+    0   0   102     102     libsoc.a
+    4   1   0   5   libwpa2.a
+    0   1   0   1   libwps.a
+======================================================
+  "Select build Option"
+======================================================
+  "1. Build with Current Configurations"
+  "2. Re-configure"
+  "3. Menuconfig"
+  "4. Build Clean"
+  "5. Build Dist-Clean"
+  "6. Build SmartFS Image"
+  "d. Download"
+  "x. Exit"
+======================================================
+d               // Select "Download"
+==================================================
+  "Select download option"
+==================================================
+  "1. ALL"
+  "2. OS"
+  "4. BOOTLOADER"
+  "5. ROMFS"
+  "6. ERASE_ALL"
+  "u. USBrule"
+  "x. Exit"
+==================================================
+1               // Select "All"
+ ALL
+make -C ../build/tools/esp32  ALL
+...
+Leaving...
+Hard resetting via RTS pin...
+make[1]: Leaving directory '/home/tizenrt/ws/public/priv/TizenRT/build/tools/esp32'
+
+                // DONE
+```
+Now, the device is ready onboarding.
+
+### Try to do EasySetup and Control   
+You can make connection between mobile and your device via easysetup refer below. and you can download [SmartThings Application](https://play.google.com/store/apps/details?id=com.samsung.android.oneconnect) via Google play app store.
+![Workflow to develop STDK Things Device](./media/STDK_EasySetup.png)  
+Now you can control smart led device(sample) through SmartThings mobile application.
+
+## How to develop your STDK device   
+You can see more detail guild in ['Getting started'](https://github.com/SmartThingsCommunity/st-device-sdk-c-ref/blob/master/doc/getting_started.md).
```

Comment:
"more detail guild" -- "a more detailed build approach in..."

Reply:
Has changed.